src/sagemaker/jumpstart/hub/hub.py

```diff
@@ -132,17 +92,20 @@ def create(
         tags: Optional[str] = None,
     ) -> Dict[str, str]:
         """Creates a hub with the given description"""
-
-        create_hub_bucket_if_it_does_not_exist(
-            self.hub_storage_location.bucket, self._sagemaker_session
-        )
+        curr_timestamp = datetime.now().timestamp()
 
         return self._sagemaker_session.create_hub(
             hub_name=self.hub_name,
             hub_description=description,
             hub_display_name=display_name,
             hub_search_keywords=search_keywords,
-            s3_storage_config={"S3OutputPath": self.hub_storage_location.get_uri()},
+            s3_storage_config={
```

Comment:
can we make a dedicated function to get the s3 output path?

Reply:
We could, though I don't think it's necessary as this logic is simple. I unit test both logic paths